$NetBSD: patch-setup.cfg,v 1.3 2024/02/07 06:25:43 adam Exp $

setuptools_scm_git_archive is part of setuptools_scm as of 7.0.0.

--- setup.cfg.orig	2023-05-20 13:33:03.848234000 +0000
+++ setup.cfg
@@ -54,8 +54,7 @@ python_requires = >=3.6
 packages = find:
 include_package_data = True
 setup_requires = 
-	setuptools_scm>=1.15.0
-	setuptools_scm_git_archive>=1.0
+	setuptools_scm>=7.0.0
 install_requires = 
 	importlib_metadata; python_version < '3.8'
 	more_itertools >= 2.6
